plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -39,6 +39,9 @@ type NodeInfo struct {
 	requestedResource *Resource
 	pods              []*api.Pod
 	nonzeroRequest    *Resource
+
+	// FIXME: Comment.
+	generation int64
```

Comment:
Yes - there is a concept of generation (which is basically exactly what we have here).
See:
https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L102